tests/test_hashing.py

```diff
@@ -9,7 +9,7 @@
 from pytest import raises
 from utils import authenticate, init_app_with_options
 
-from flask_security.utils import encrypt_password, verify_password
+from flask_security.utils import hash_password, verify_password
```

Comment:
I would propose to keep the tests untouched until we completely remove the deprecated version. WDYT?